androidApp/src/main/java/dev/johnoreilly/confetti/search/SearchRoute.kt

```diff
@@ -21,8 +24,9 @@ fun SearchRoute(
     onSignOut: () -> Unit,
 ) {
     val viewModel = getViewModel<SearchViewModel>()
+    val user by koinInject<Authentication>().currentUser.collectAsStateWithLifecycle()
     SideEffect {
-        viewModel.configure(conference)
+        viewModel.configure(conference, user?.uid, user)
```

Comment:
Just wondering, Is there a case where the uid is different from the user.uid? :)

Reply:
Never 😅 . It's just that `User` is not available in `commonMain`, only in `mobileMain` so we can't pass it all the way down. I guess it also prevents leakage of `photoUrl`, `name` and `email` which aren't really needed but we could certainly put everything in only one parameter at some point.

Reply:
Aha okay, yeah I was just a bit confused since you pass both here :)